cmd/command.go

```diff
@@ -21,8 +21,10 @@ import (
 	"github.com/superfly/flyctl/terminal"
 )
 
+// CmdRunFn - Run function for commands which takes a command context
 type CmdRunFn func(*CmdContext) error
 
+// Command - Wrapper for a cobra command
```

Comment:
Nice improvement!

Reply:
I was just getting bugged by all the yellow in vscode's view. :)